packages/js/product-editor/src/blocks/inventory-quantity/block.json

```diff
@@ -1,11 +1,11 @@
 {
 	"$schema": "https://schemas.wp.org/trunk/block.json",
 	"apiVersion": 2,
-	"name": "woocommerce/product-track-inventory-fields",
-	"title": "Product track inventory fields",
+	"name": "woocommerce/product-inventory-quantity-field",
```

Comment:
You forgot to change this also in https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/src/Admin/Features/ProductBlockEditor/BlockRegistry.php#L39